backend/src/controllers/squad.js

```diff
@@ -0,0 +1,101 @@
+exports.getSquad = async (req, res, next) => {  
+  try {
+    // Fetch squad from database
+    const squad = await req.context.models.Squad.findByPk(req.params.squadId);
+    const squadData = squad.dataValues;
+  
+    // Return squad
+    res.status(201);
+    res.send(squadData);
+  } catch (err) {
+    console.log(err)
+    res.status(404);
+    res.send({ error: "Could not find squad." });
```

Comment:
I wasn't doing this either, but to make it a little cleaner we could fit these onto one line ([source](https://stackoverflow.com/questions/26066785/proper-way-to-set-response-status-and-json-content-in-a-rest-api-made-with-nodej))

```suggestion
    res.status(404).send({ error: "Could not find squad." });
```